windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -17,6 +17,36 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
 
 You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
 
+[id="wmco-2-0-1"]
+== Release notes for Red Hat Windows Machine Config Operator 2.0.1
+
+Issued:2021-06-23
+
+The WMCO 2.0.1 is now available with bug fixes. The components of the WMCO were released in [].
```

Comment:
Not sure if I just don't understand the docs syntax, but seems like there is something missing at the end of the sentence `in []`?

Reply:
Yup! In the [1.0.4 documentation set](https://github.com/openshift/openshift-docs/pull/31531) we [linked to this](https://access.redhat.com/errata/RHBA-2021:1061). I tried to manually find a similar type of link, but no luck.

Reply:
You can find this on the [errata](https://errata.devel.redhat.com/advisory/76595) associated with the release under `Public URL` in the `Information` section: https://access.redhat.com/errata/RHSA-2021:2130

Reply:
Here it is for 2.0.1: https://access.redhat.com/errata/RHSA-2021:2130